ethsigner/subcommands/src/main/java/tech/pegasys/ethsigner/subcommands/AzureSubCommand.java

```diff
@@ -96,6 +94,33 @@ private TransactionSigner createSigner() throws TransactionSignerInitializationE
     return factory.createSigner(config);
   }
 
+  @Override
+  protected void validateArgs() throws InitializationException {
+    super.validateArgs();
```

Comment:
The `SignerSubCommand` which is the super class of all subcommands, has validateArgs, which call `EthSignerBaseCommand`'s validateArgs (which performs validation on the cli options of the main command).

IFF we don't have subcommands and all cli options exist in EthSignerBaseCommand, then all of this will go away.